tests/python/unittest/test_ir_container.py

```diff
@@ -91,5 +125,10 @@ def test_ndarray_container():
     test_map()
     test_array_save_load_json()
     test_map_save_load_json()
+    test_dir_array()
+    test_dir_map()
+    test_getattr_array()
+    test_getattr_map()
```

Comment:
Would you mind to help change to the pytest style?
```python
if __name__ == "__main__":
    pytest.main([__file__])
```

Reply:
Thank you for your valuable advice! I have rewritten these four test cases and changed the test file to the pytest style.

Reply:
Could you help review this new commit as it passes all checks :-D @comaniac